addon/components/row/loading.ts

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import { readOnly } from '@ember/object/computed';
+// @ts-ignore: Ignore import of compiled template
+import layout from '@gavant/ember-table/templates/components/row/loading';
+
+export default class RowLoadingComponent extends Component {
+    layout = layout;
+    tagName: string = 'tr';
+    classNames: string[] = ['et-tr', 'data-table-row-loading'];
+    cellClassNames: string = 'p-4';
+
+    //readonly attributes
+
+    @readOnly('columns.length') colspan?: number;
```

Comment:
@napafundi why would `colspan` be optional?

Reply:
Wasn't sure how to properly handle these types of values in TS. There are two options recommended. 1) adding `| undefined` to the declaration, and 2) adding `!:` definite declaration. Since the columns value is guaranteed, I'm guessing the definite declaration is the way to go?

Reply:
yeah i think we've gone with the `!` approach in other cases like this before (such as when defining types for injected services), because we know that that computed macro will always return a value.